### PR TITLE
Refine CiV documents

### PR DESCRIPTION
**Pull request by YadongQi**
For latest release, start_android_qcow2.sh is replaced by start_civ.sh,
so update the document accordingly.

Signed-off-by: Qi, Yadong <yadong.qi@intel.com>

### CODE REVIEW

**Comment by YadongQi** [comment]
@ttzeng Please help review this PR. Thanks.

File: source/getting-started/on-vm.rst [hunk 1]
```diff
@@ -317,7 +345,7 @@ by adding :command:`--audio-passthrough` argument while invoking the
 
 .. _QEMU: https://www.qemu.org/
 
-.. _start_android_qcow2.sh: https://raw.githubusercontent.com/projectceladon/device-androidia-mixins/master/groups/device-specific/caas/start_android_qcow2.sh
+.. _start_civ.sh: https://github.com/projectceladon/vm_manager/raw/master/scripts/start_civ.sh
```
Comment (ttzeng) [inline source/getting-started/on-vm.rst:348]: I think users no longer require to download the scripts separately, we can remove this link.

File: source/getting-started/on-vm.rst [hunk 2]
```diff
@@ -165,64 +146,127 @@ a *qcow2* formatted virtual disk.
    Now we get the CiV virtual disk :file:`android.qcow2` under the current
    directory.
 
-Reboot to Android UI
+Boot to Android UI
 ********************
```
[the remark this answers is not in the document]
Comment (YadongQi) [reply source/getting-started/on-vm.rst]: done

File: source/getting-started/on-vm.rst (hunk 1 above)
Comment (YadongQi) [reply source/getting-started/on-vm.rst]: done

File: source/getting-started/on-vm.rst [hunk 3]
```diff
@@ -165,64 +146,127 @@ a *qcow2* formatted virtual disk.
...
 .. code-block:: bash
 
     # The following command runs CiV using Intel GVT-d, and passes
     # all the attached USB devices such as keyboard, mouse to the VM.
-    $ sudo -E ./scripts/start_android_qcow2.sh --gvtd --usb-host-passthrough
+    $ sudo -E ./scripts/start_civ.sh -g GVT-d --passthrough-pci-usb
 
-xHCI pass-through option
+USB PCI controller pass-through option
 ========================
```
[the remark this answers is not in the document]
Comment (YadongQi) [reply source/getting-started/on-vm.rst]: done

**Pull request merged**